src/app/index.tsx

```diff
@@ -36,10 +36,14 @@ const AppRoutes = (): React.ReactElement => (
       <Route path="accounts/:accountName" element={<AccountDetails />} />
       <Route path="clusters" element={<Clusters />} />
       <Route path="clusters/:clusterID" element={<ClusterDetails />} />
-      <Route path="servers" element={<Servers />} />
-      <Route path="servers/:instanceID" element={<ServerDetails />} />
-      <Route path="observe/audit-logs" element={<AuditLogs />} />
-      <Route path="observe/scheduler" element={<Scheduler />} />
+      <Route path="instances" element={<Servers />} />
+      <Route path="instances/:instanceID" element={<ServerDetails />} />
+      {/* Scan routes */}
+      <Route path="scan/scheduler" element={<Scheduler />} />
+      <Route path="scan/audit-logs" element={<AuditLogs />} />
+      {/* Actions routes */}
+      <Route path="actions/scheduler" element={<Scheduler />} />
+      <Route path="actions/audit-logs" element={<AuditLogs />} />
```

Comment:
As we discussed in our last meeting, please remove the "Scan/Audit-logs".

Audit logs are only part of "Actions" menu